src/System.Buffers/ref/System.Buffers.csproj

```diff
@@ -1,15 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
   	<!-- Must match version supported by frameworks which support 4.0.* inbox.
          Can be removed when API is added and this assembly is versioned to 4.1.* -->
     <AssemblyVersion>4.0.2.0</AssemblyVersion>
     <ProjectGuid>{11AE73F7-3532-47B9-8FF6-B4F22D76456C}</ProjectGuid>
-    <Configurations>netstandard-Debug;netstandard-Release;netcoreapp-Debug;netcoreapp-Release;uap-Debug;uap-Release</Configurations>
+    <Configurations>netcoreapp-Debug;netcoreapp-Release;uap-Debug;uap-Release</Configurations>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Buffers.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'uap' OR '$(TargetGroup)' == 'netcoreapp'">
+  <ItemGroup>
     <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
+    <ProjectReference Include="..\..\System.Memory\ref\System.Memory.csproj" />
```

Comment:
cc @ericstj, can you please review the props/csproj changes related to System.Buffers? I removed the `netstandard` configuration since we only build this library in-box now, targeting .NET Core.

Reply:
It turns out I can't remove netstandard so conditionally including the new APIs.

Reply:
Alternatively, we need to keep building System.Memory for `netstandard`

Reply:
No, you don't need to.  You can use a simple-name reference to get the version we restore from packages.

Reply:
Since you aren't shipping the netstandard config nor changing it removing it is correct.  You should ensure that you harvest and bin place the old assembly (so that folks don't build against a new version that doesn't ship).